packages/table/src/TableContainer/TableContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { forwardRef, PropsWithChildren } from 'react';
+
+import { css, cx } from '@leafygreen-ui/emotion';
+
+import TableContainerProps from './TableContainer.types';
+
+const TableContainer = forwardRef<HTMLDivElement, TableContainerProps>(
+  (
+    { children, className, ...rest }: PropsWithChildren<TableContainerProps>,
+    forwardedRef,
+  ) => {
+    return (
+      <div
+        ref={forwardedRef}
+        className={cx(
+          css`
+            overflow: auto;
+            height: 500px;
```

Comment:
where does 500px come from?

Reply:
Arbitrary, but the TableContainer needs a height or max-height set to make `position: sticky` work on the header. If developers can change this pretty easily using the `className` if they want, so I didn't think this would be a problem.

Reply:
Actually, `max-height` is better here. Gonna change this